pdf_annotate/graphics.py

```diff
@@ -142,7 +143,7 @@ def resolve(self):
 
 class Text(namedtuple('Text', ['text']), NoOpTransformBase):
     def resolve(self):
-        return '({}) Tj'.format(self.text)
+        return PdfString.encode(self.text) + ' Tj'
```

Comment:
I originally did this by hand until I saw pdfrw had a function to encode strings already, one caveat is that ascii strings with non-displayable characters will get encoded with \xDD now which is different than before.  But this should all conform to PDF standards and work correctly.